force-app/main/selector/ContactSelector.cls

```diff
@@ -62,6 +62,25 @@ public inherited sharing class ContactSelector {
         return Database.Query(householdMembersQueryFor(queryHHIds));
     }
 
+    /*********************************
+     * @description Returns a Contact with the donations summary
+     * @param contactId Id of the contact
+     * @returns Returns a Contact with the donations summary
```

Comment:
method description generally is not required anymore because the method name and return type should be explanatory. If the header is kept, the `@returns` section should be rewritten as `@return Contact`